db/python/layers/sequence.py

```diff
@@ -188,6 +188,16 @@ async def update_sequencing_status_from_internal_sample_id(
         seq_id = self.get_latest_sequence_id_for_sample_id(sample_id)
         return self.update_status(seq_id, status, check_project_id=False)
 
+    async def update_sample_sequence_status_by_type(
+        self, sample_id: int, status: SequenceStatus, sequence_type: SequenceType
+    ):
+        """Update the sequencing status of a given type from the internal sample id"""
+        seq_id = await self.seqt.get_latest_sequence_id_by_type(
+            sample_id, sequence_type
+        )
+
+        return await self.update_status(seq_id, status, check_project_id=False)
```

Comment:
Should check the project ID before, esp if you're asking to skip the project check on update status